### PR TITLE
feat(web): show exchange arrows after closing side panel

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Mobile exchange arrows are not present when navigation back from the side panel

## Description
This PR updates the map moving atom when navigating back so the exchange arrows become visible again after navigating back from the full screen side panel

### CODE REVIEW

File: web/src/features/panels/LeftPanel.tsx [hunk 1]
```diff
@@ -97,9 +97,10 @@ function MobileHeader() {
 
 function OuterPanel({ children }: { children: React.ReactNode }) {
   const [isOpen, setOpen] = useAtom(leftPanelOpenAtom);
-  const onCollapse = () => setOpen(!isOpen);
   const location = useLocation();
 
+  const onCollapse = () => setOpen(!isOpen);
+
```
Comment (VIKTORVAV99) [inline web/src/features/panels/LeftPanel.tsx:103]: Im curious, doe the location of this matter? or was it just a preference change?

**Review by VIKTORVAV99 — APPROVED** [review]
Looks good just one small question really.

File: web/src/features/panels/LeftPanel.tsx (hunk 1 above)
Comment (tonypls) [reply web/src/features/panels/LeftPanel.tsx:103]: Doesn't matter 😆 I originally thought the fix might come from this file so I had changed it a bit. My ordering logic was bunch together variable definitions then have a line between functions defined within a component

**Review by silkeholmebonnen — APPROVED** [review]
LGTM!

**Pull request merged**